cluster/manifests/prometheus/rbac.yaml

```diff
@@ -8,6 +8,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: prometheus
+  labels:
+    admission.zalando.org/infrastructure-component: "true"
```

Comment:
Is there some particular reason we're doing this specifically for prometheus only? I'm assuming given the rules of admission specified above, we will need this for all other infrastructure resources as well?

(Like say, the RBAC resources in the `roles` directory)

I know the feature is disabled by default so it's not relevant, but I'm curious, in terms of enabling this long term

Reply:
Yes, it will be needed on all resources but I'm not sure yet if it's the right approach. I didn't want to find and attach it to everything yet to keep the diff small. On the other hand I want something I can test with.

Reply:
Makes sense. I also saw in your [comment on the other PR](https://github.com/zalando-incubator/kubernetes-on-aws/pull/8517/files#r1831487728) that's to trigger an opt-in where we can test. Sounds good 🙂